app/src/main/java/org/sopt/soptseminar_week1/view/FollowingListAdapter.kt

```diff
@@ -1,28 +1,24 @@
 package org.sopt.soptseminar_week1.view
 
 import android.content.Context
-import android.util.Log
 import android.view.LayoutInflater
 import android.view.ViewGroup
 import androidx.recyclerview.widget.RecyclerView
 import com.bumptech.glide.Glide
-import org.sopt.soptseminar_week1.data.FollowingListUserInfo
+import org.sopt.soptseminar_week1.data.GithubUserInfo
 import org.sopt.soptseminar_week1.databinding.ItemFollowingListBinding
 
-class FollowingListAdapter : RecyclerView.Adapter<FollowingListAdapter.FollowingUserViewHolder>() {
+class FollowingListAdapter(followings: List<GithubUserInfo>) :
+    RecyclerView.Adapter<FollowingListAdapter.FollowingUserViewHolder>() {
 
-    private val userList = mutableListOf<FollowingListUserInfo>()
-
-    fun setUserList(newList : List<FollowingListUserInfo>) {
-        userList.addAll(newList)
-    }
+    private val userList = followings.toList()
 
     class FollowingUserViewHolder(private val binding: ItemFollowingListBinding) :
         RecyclerView.ViewHolder(binding.root) {
-        fun onBind(followingListUserInfo: FollowingListUserInfo, context: Context) {
-            binding.textFollowingListProfile.text = followingListUserInfo.userName
-            Glide.with(context).load(followingListUserInfo.userImageSrc).into(binding.imgFollowingListProfile)
-            Log.d("로그", "리사이클러뷰 만들어짐 ${followingListUserInfo.userImageSrc}")
+        fun onBind(followingListUserInfo: GithubUserInfo, context: Context) {
+            binding.textFollowingListProfile.text = followingListUserInfo.login
+            Glide.with(context).load(followingListUserInfo.avatar_url)
+                .into(binding.imgFollowingListProfile)
```

Comment:
Context가 있다는 것은 이 recyclerview는 view를 바인딩해주는 역할뿐만 아니라 다른 역할도 수행할 수도 있다는 거 아닐까? 
```suggestion
        fun onBind(followingListUserInfo: GithubUserInfo) {
            binding.textFollowingListProfile.text = followingListUserInfo.login
            Glide.with(binding.imgFollowingListProfile.context).load(followingListUserInfo.avatar_url)
                .into(binding.imgFollowingListProfile)
```
Glide 때문에 context가 필요한거라면 이정도면 충분하다고 생각해

Reply:
오 여기 많이 고민했었는데 진짜 고맙다!!